version.sh

```diff
@@ -76,3 +76,40 @@ for i in "${!module_names[@]}"; do
 done
 
 echo -e "$res" | column -t -s '|'
+
+echo
+echo "*** Creating requirements file with current emirge module versions"
+
+
+outfile=$(mktemp requirements.txt.XXXXXX)
```

Comment:
Hmm, I don't disagree with the flag, but it's not clear to me what you mean with the output to stdout - the requirements are already printed to stdout (as well as to the file). Should I remove the printing of the warnings? Or remove the printing to the file? Imho, creating a file with the requirements directly can be useful, since it allows the automated recording of some meta data (like the time stamp that is currently recorded).

Reply:
Print to file is fine. Stdout is just more composable for shell operations, but I don't think it matters. Diagnostic stuff (such as warnings) should go to stderr anyway.

Reply:
As it is, this script is not usable by another script (without grepping the output for the generated filename). It should be.

Reply:
I've changed it so the user needs to specify a filename.